File: Programmers/Lv3/Lv3_숫자게임.cpp
#include <algorithm>
#include <vector>
using namespace std;

int solution105(vector<int> A, vector<int> B) {
	int answer = 0;
	sort(A.begin(), A.end());
	sort(B.begin(), B.end());
	int save = 0;
	
	for (int i = 0; i < A.size(); i++) {
		for (int j = save; j < B.size(); j++) {
			if(A[i]<B[j]){
				answer++;
				save = j;
				save++;
				break;
			}
		}
	}
	return answer;
}

File: Programmers/Lv3/Lv3_숫자게임.py
def solution(A, B):
    answer = 0
    A.sort()
    B.sort()
    save = 0

    for i in range(len(A)):
        for j in range(save, len(B)):
            if A[i] < B[j]:
                answer += 1
                save = j
                save += 1
                break

    return answer